tests/Behat/Service/ElasticsearchCommands.php

```diff
@@ -95,14 +94,14 @@ private function populateIndexType(
         array $options
     ): void
     {
-        $event = new TypePopulateEvent($index, $type, $reset, $options);
-        $this->dispatcher->dispatch($event, TypePopulateEvent::PRE_TYPE_POPULATE);
+        $event = new PostIndexPopulateEvent($index, $reset, $options);
+        $this->dispatcher->dispatch($event, 'elastica.index.type_pre_populate');
```

Comment:
As FOS is no longer providing const with those names I think we should create them on our own. If const is going to be used only in one class you can declare it on a class level otherwise create a separate file.